src/internal.c

```diff
@@ -25,6 +25,7 @@
     #include <config.h>
 #endif
 
+#include <wolfssl/wolfcrypt/wc_port.h>
```

Comment:
Why do we need wc_port.h included here?  Shouldn't it already be included since internal.h includes types.h, which then includes wc_port.h?